components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/docs/examples/alerts/alerts.yaml

```diff
@@ -0,0 +1,48 @@
+get:
+  - request:
+      method: GET
+      url: https://localhost:9443/api/am/store/v1.0/alert-types
+      headers: |
+          Authorization: Bearer eff01a97-4dc0-3542-a184-75aecf355ec2
+    response:
+      status:
+        code: 200
+        msg: OK
+      headers: |
+          Content-Type: application/json
+      body:
+        {
+          "count":2,
+          "alerts":[
+          {
+            "id":3,
+            "name":"AbnormalRequestsPerMin",
+            "isRequireConfiguration":true
+          },
+          {
+            "id":4,
+            "name":"AbnormalRequestPattern",
+            "isRequireConfiguration":false
+          }]
+        }
+  - rawRequest: |
```

Comment:
Let's remove these as mentioned above.

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/7117

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/7117